src/Providers/UserProviders.jsx

```diff
@@ -1,10 +1,14 @@
 import React, { Component, createContext } from "react";
 import { auth, generateUserDocument } from "../firebaseCredentials.js";
+import { httpGet } from "../lib/dataAccess";
+
+const baseEndpoint = process.env.REACT_APP_ADMIN_BASE_ENDPOINT;
 
 export const UserContext = createContext({ user: null });
 class UserProvider extends Component {
     state = {
         user: null,
+        isAdmin: false,
     };
```

Comment:
Any plan on refactoring this to functional components in the future?

Reply:
Hmm, not high priority. Class + functional components work together as far as I know